tests/new_functional/pages/surveys/question.page.js

```diff
@@ -0,0 +1,25 @@
+class QuestionPage {
+
+  constructor(pageName) {
+    this.pageName = pageName;
+    this.questions = [];
+  }
+
+  alert() { return '[data-qa="error-body"]';  }
+
+  error() { return '.js-inpagelink'; }
+
+  previous() { return 'a[id="top-previous"]'; }
+
+  displayedName() { return 'h1[class="section__title saturn"]'; }
+
+  submitButton() { return '[data-qa="btn-submit"]'; }
```

Comment:
Given the amount this is used I think it would make sense to shorten it to `submit()`